scss/bootstrap/overrides/crm/_error.scss

```diff
@@ -1,12 +1,9 @@
 .crm-error {
   border-radius: $border-radius-small;
+  display: inline-block;
+  margin-top: $crm-btn-padding-small-vertical;
```

Comment:
margin-top: $...padding...

Can we simply change the variable name from -padding- to something else? Maybe `$crm-gap-small` or smth like that?

What do you think?

Reply:
Agreed, though we'll need to add that to the general variables.